modules/internal/ChapelArray.chpl

```diff
@@ -1030,6 +1030,16 @@ module ChapelArray {
       _value.dsiAdd(i);
     }
 
+    proc bulkAdd(inds: [] _value.idxType, sorted=false,
+        noDuplicate=false) where isSparseDom(this) && _value.rank==1 {
```

Comment:
I like sorted=false but I'd consider renaming 'noDuplicate' to 'noDuplicates' or 'unique'